life_expectancy/cleaning.py

```diff
@@ -1,83 +1,120 @@
-"""Module to clean dataframe"""
-import sys
-from typing import List
+"""Cleaning file with objects to clean various types of file extensions"""
```

Comment:
```suggestion
"""
Module for file cleaning. 
Supported file extensions: .json, .csv
"""
```
It would be more informative.